src/content/guides/getting-started.md

```diff
@@ -178,25 +179,25 @@ __dist/index.html__
 
 In this setup, `index.js` explicitly requires `lodash` to be present, and binds it as `_` (no global scope pollution). By stating what dependencies a module needs, webpack can use this information to build a dependency graph. It then uses the graph to generate an optimized bundle where scripts will be executed in the correct order.
 
-With that said, let's run `npx webpack` with our script as the [entry point](/concepts/entry-points) and `bundle.js` as the [output](/concepts/output). The `npx` command, which ships with Node 8.2 or higher, runs the webpack binary (`./node_modules/.bin/webpack`) of the webpack package we installed in the beginning:
+With that said, let's run `npx webpack` with our script as the [entry point](/concepts/entry-points) and `main.js` as the (default) [output](/concepts/output). The `npx` command, which ships with Node 8.2 or higher, runs the webpack binary (`./node_modules/.bin/webpack`) of the webpack package we installed in the beginning:
```

Comment:
why do we need parentheses here `(default)`?

Reply:
It was to clarify the statement a little - it says to run the command with main.js as the output (and I suppose src/index.js as the input!), but the actual command invoked is just `webpack`, and you're not actually doing anything to cause the output file to be main.js, you're just running the command and it defaults that. Perhaps re-writing that would read more easily, something like this?

"With that said, let's run `npx webpack` with no arguments, which will take our script at `src/index.js` as the [entry point](/concepts/entry-points), and will generate `dest/main.js` as the [output](/concepts/output)."